dev_tools/Dockerfile-backup

```diff
@@ -0,0 +1,22 @@
+FROM golang:1.16-alpine
+WORKDIR /source
+
+COPY go.mod ./
+COPY go.sum ./
+
+RUN go mod download
+
+COPY *.go ./
+RUN CGO_ENABLED=0 go build -o /bin/sda-backup
+
+FROM postgres:13-alpine
+ARG USER_ID
+ARG GROUP_ID
+
+ENV POSTGRES_PASSWORD random-pw
+COPY dev_tools/config_postgres.yaml ./
+COPY dev_tools/certs ./dev_tools/certs
+COPY dev_tools/keys ./dev_tools/keys
+COPY --from=0 /bin/sda-backup /bin/sda-backup
+ENV CONFIGFILE="../config_postgres.yaml"
+ENV UID=$USER_ID
```

Comment:
This container should no be needed

Reply:
The integrity run only for postgres 13 and I was facing some compatibility issues (for example it was impossible to run from my laptop) and in order to avoid those issues I made this container.